### PR TITLE
Improve RuntimeScene typing

### PR DESCRIPTION
**Pull request by arthuro555**
Improve typing in RuntimeScene

### CODE REVIEW

File: GDJS/Runtime/runtimescene.ts [hunk 1]
```diff
@@ -1078,13 +1078,15 @@ namespace gdjs {
...
 
-    //The flags to describe the change request by a scene:
-    static CONTINUE = 0;
-    static PUSH_SCENE = 1;
-    static POP_SCENE = 2;
-    static REPLACE_SCENE = 3;
-    static CLEAR_SCENES = 4;
-    static STOP_GAME = 5;
+  //The flags to describe the change request by a scene:
+  export enum SceneChangeRequest {
```
Comment (4ian) [inline GDJS/Runtime/runtimescene.ts:1084]: Can you check how this enum is generated by esbuild? :)

**Review by 4ian — APPROVED** [review]
Looks fair overall!

File: GDJS/Runtime/runtimescene.ts (hunk 1 above)
Comment (arthuro555) [reply GDJS/Runtime/runtimescene.ts:1084]:
```js
gdjs2.RuntimeScene = RuntimeScene;
  let SceneChangeRequest;
  (function(SceneChangeRequest2) {
    SceneChangeRequest2[SceneChangeRequest2["CONTINUE"] = 0] = "CONTINUE";
    SceneChangeRequest2[SceneChangeRequest2["PUSH_SCENE"] = 1] = "PUSH_SCENE";
    SceneChangeRequest2[SceneChangeRequest2["POP_SCENE"] = 2] = "POP_SCENE";
    SceneChangeRequest2[SceneChangeRequest2["REPLACE_SCENE"] = 3] = "REPLACE_SCENE";
    SceneChangeRequest2[SceneChangeRequest2["CLEAR_SCENES"] = 4] = "CLEAR_SCENES";
    SceneChangeRequest2[SceneChangeRequest2["STOP_GAME"] = 5] = "STOP_GAME";
  })(SceneChangeRequest = gdjs2.SceneChangeRequest || (gdjs2.SceneChangeRequest = {}));
  ```

Comment (4ian) [reply GDJS/Runtime/runtimescene.ts:1084]: So it's generating "two ways" enum between integers and a string. Fair enough.

Comment (arthuro555) [reply GDJS/Runtime/runtimescene.ts:1084]: > gdjs2?

It is the way esbuild handles exports, it wraps it all in an IIFE like this:
```js
let gdjs;
(function(gdjs2) {
    gdjs2.something = 1;
})(gdjs | (gdjs = {}))
```

**Pull request merged**